ng-dev/commit-message/parse.ts

```diff
@@ -110,7 +110,7 @@ const parseOptions: Options & {notesPattern: (keywords: string) => RegExp} = {
   headerPattern,
   headerCorrespondence,
   noteKeywords: [NoteSections.BREAKING_CHANGE, NoteSections.DEPRECATED],
-  notesPattern: (keywords: string) => new RegExp(`^\s*(${keywords}): ?(.*)`),
+  notesPattern: (keywords: string) => new RegExp(`^(${keywords}): ?(.*)`),
```

Comment:
We should still allow for superfluous space and just add the backslash to actually use the special `\s` regex syntax

```suggestion
  notesPattern: (keywords: string) => new RegExp(`^\\s*(${keywords}): ?(.*)`),
```